CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Changelog
+
+## 1.3.0 - 2024-08-14
+
+1. Added the ability to explicitly specify the uuid for capturing events. If uuid is not specified, Posthog will calculate it automatically (suitable for most cases).
+2. Removed unused config function `uid()`.
+
```

Comment:
@evgeek this file is auto-generated, you don't need to make edits to it.

Reply:
+